ckanext/canada/tables/qpnotes.yaml

```diff
@@ -0,0 +1,357 @@
+
+  examples:
+    record:
+      reference_number: "DND-2019-QP-00001"
+      title_en: "Bill C-58 (Access to Information Reform)"
+      title_fr: "Projet de loi C-58 (réforme de l'accès à l'information)"
+      minister: "Scott Brison"
+      question_en: "What is the Government doing to reform the Access to Information Act (Bill C-58)?"
+      question_fr: "Que fait le gouvernement pour réformer la Loi sur l'accès à l'information (projet de loi C-58)?"
+      date_received: "2018-02-15"
+      background_en: "Government of Canada institutions subject to the Access to Information Act <linkto:http://laws-lois.justice.gc.ca/eng/acts/a-1/> (ATIA) are required to post summaries of processed ATI requests."
+      background_fr: "Les institutions du gouvernement du Canada visées par la Loi sur l’accès à l’information <linkto : http://laws-lois.justice.gc.ca/fra/lois/a-1/> (LAI) sont tenues de publier les sommaires des demandes d’accès à l’information qu’elles ont traitées."
+      response_en: "• Bill C-58 contains the first significant amendments to the Access to Information Act since it was introduced over three decades ago.
+       \n• This Bill is the first phase of a two-step approach to update access to information. The targeted changes will be followed by a full review of the Act in Phase II, to begin within one year of royal assent.
+       \n• The Government thanks the members of the Senate Committee on Legal and Constitutional Affairs for their careful review of Bill C-58."
+      response_fr: "• Le projet de loi C-58 contient les premières modifications importantes touchant la Loi sur l’accès à l’information depuis l’adoption de celle-ci il y a plus de trente ans.
+       \n• Le projet de loi C-58 est la première phase d’une approche en deux étapes qui vise à mettre à jour l’accès à l’information. Les changements visés seront suivis d’un examen complet de la Loi au cours de la deuxième phase, qui commencera dans l’année suivant la sanction royale du projet de loi.
+       \n• Le gouvernement remercie les membres du Comité sénatorial permanent des affaires juridiques et constitutionnelles pour l’examen approfondi du projet de loi C-58 qu’ils ont effectué."
+      additional_information_en: "The Government will continue to work with the Information Commissioner, parliamentarians, and other stakeholders to further strengthen government openness and transparency."
+      additional_information_fr: "Le gouvernement continuera de travailler avec la commissaire à l’information, les parlementaires et d’autres intervenants afin d’être encore plus ouvert et transparent. "
+    filter_one:
+      reference_number: DND-2019-QP-00001
+    sort: date_received desc
+
+excel_edge_style:
+  PatternFill:
+    fgColor: FF336B87
+    patternType: solid
+excel_header_style:
+  PatternFill:
+    patternType: solid
+    fgColor: FF90AFC5
+excel_column_heading_style:
+  PatternFill:
+    patternType: solid
+    fgColor: FF90AFC5
```

Comment:
Have you chosen some nice header+edge colours for this template? This is where we get to make the template stand out from the others :-)